src/CarModel.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace MyStore;
+
+use PDO;
+
+class CarModel
+{
+    public PDO $db;
+
+    public function __construct(PDO $db)
+    {
+        $this->db = $db;
+    }
+
+    public function getProperties(): array | false
```

Comment:
The return type hint is not accurate, unlike fetch, fetchAll always returns an array, even if there are no results (empty array)

Reply:
Also I'd suggest a more semantic name, something like getAllCars()